examples/demo-app/src/app.js

```diff
@@ -66,6 +66,110 @@ import sampleIconCsv, {config as savedMapConfig} from './data/sample-icon-csv';
 import {processCsvData, processGeojson} from 'kepler.gl/processors';
 /* eslint-enable no-unused-vars */
 
+const mockData = {
+  graph: {
+    label: '8672f0ad-ce27-488a-b477-b64a4c57be41',
+    type: 'simulation',
+    nodes: [
+      {
+        id: '1',
+        label: 'Power plant',
+        metadata: {
+          'is-a': ['room'],
+          gis: [],
+          x: 4.434661720198483,
+          y: 51.20594386351399,
+          'water-level-for-flooding-in-cm': [23],
+          icon: 'https://stcitiespublic.blob.core.windows.net/assets/precinct/marker.png',
```

Comment:
Always use our cdn endpoint
```suggestion
          icon: 'https://cdne-cities-assets.azureedge.net/precinct/marker.png',
```